smdebug/core/writer.py

```diff
@@ -155,6 +156,7 @@ def write_tensor(
         tag = tname
         tensor_proto = make_tensor_proto(nparray_data=value, tag=tag)
         s = Summary(value=[Summary.Value(tag=tag, metadata=smd, tensor=tensor_proto)])
+        self.logger.debug(f"smdebug: writing tensor: {tname}")
```

Comment:
Shouldn't the logger itself handle prepending smdebug as an identifier to each logging message?